notNeededPackages.json

```diff
@@ -7987,6 +7991,10 @@
             "libraryName": "webmidi",
             "asOfVersion": "3.0.1"
         },
+        "webpack": {
+            "libraryName": "webpack",
+            "asOfVersion": "5.28.6"
+        },
```

Comment:
I feel pretty scared of this. When this is published, it'll make `@types/webpack` at v5, containing `"webpack": "*"`.

Perhaps that is fine, given 4 is untyped but 5 is?